js/ad_manager_vast.js

```diff
@@ -561,7 +561,14 @@ OO.Ads.manager(function(_, $) {
           viewMode,
           creativeData = {};
 
+    if (_isVpaidAd(currentAd)) {
       currentAd.data = currentAd.ad.data;
+
+      var mediaFileUrl = _cleanString(currentAd.ad.mediaFile.url);
```

Comment:
Can you check (currentAd && currentAd.ad && currentAd.ad.mediaFile)?  Otherwise this could potential crash. Or you can do a try/catch